packages/p2p/src/components/app.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import { Tabs } from 'deriv-components';
+import { localize } from 'deriv-translations/lib/i18n';
+import Ads from './ads/ads.jsx';
+import Orders from './orders/orders.jsx';
+import MyAds from './my-ads/my-ads.jsx';
+import MyProfile from './my-profile/my-profile.jsx';
+import { init } from '../utils/websocket';
+import './app.scss';
+
+class App extends Component {
+
+    constructor(props) {
+        super(props);
+
+        init(this.props.websocket_api);
+
+        this.state = {
+            activeIndex: 0,
+        }
+    }
+
+    componentDidMount() {
+        const index_to_set = /orders/.test(window.location.pathname) ? 1 : 0;
```

Comment:
This will not work as intended. But we can evaluate it later. Can you please add a TODO with tag?

Reply:
sure

Reply:
done